<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelTrilho07">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: yellow</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem07">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho03">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem04">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>170</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho02">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: cyan</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem02">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho05">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: purple</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem03">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho06">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: orange
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem06">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho04">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>140</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem01">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho01">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>140</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: pink
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem05">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>(Un)Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem00">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem08">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem09">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem01">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem02">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem04">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem03">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem05">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem07">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSem06">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 8</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemText9">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Semáforo 9</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log de Eventos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Média (s):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremTime1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 4:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremTime2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremTime3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremTime4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 6:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 7:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremTime5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremTime6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremTime7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremLastTime4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremLastTime1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremLastTime2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremLastTime6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremLastTime7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremLastTime3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremLastTime5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremSD4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremSD1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremSD2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremSD6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremSD7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremSD3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTremSD5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Últ. volta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Des. Pad.:</string>
    </property>
   </widget>
   <zorder>labelTrilho07</zorder>
   <zorder>labelTrilho03</zorder>
   <zorder>labelTrilho02</zorder>
   <zorder>labelTrilho05</zorder>
   <zorder>labelTrilho06</zorder>
   <zorder>labelTrilho04</zorder>
   <zorder>labelTrilho01</zorder>
   <zorder>labelTrem03</zorder>
   <zorder>labelTrem01</zorder>
   <zorder>labelTrem07</zorder>
   <zorder>labelTrem04</zorder>
   <zorder>labelTrem02</zorder>
   <zorder>labelTrem06</zorder>
   <zorder>labelTrem05</zorder>
   <zorder>textEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>labelSem00</zorder>
   <zorder>labelSem08</zorder>
   <zorder>labelSem09</zorder>
   <zorder>labelSem01</zorder>
   <zorder>labelSem02</zorder>
   <zorder>labelSem04</zorder>
   <zorder>labelSem03</zorder>
   <zorder>labelSem05</zorder>
   <zorder>labelSem07</zorder>
   <zorder>labelSem06</zorder>
   <zorder>labelSemText0</zorder>
   <zorder>labelSemText1</zorder>
   <zorder>labelSemText2</zorder>
   <zorder>labelSemText3</zorder>
   <zorder>labelSemText4</zorder>
   <zorder>labelSemText5</zorder>
   <zorder>labelSemText6</zorder>
   <zorder>labelSemText7</zorder>
   <zorder>labelSemText8</zorder>
   <zorder>labelSemText9</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>boxTremTime1</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>boxTremTime2</zorder>
   <zorder>boxTremTime3</zorder>
   <zorder>boxTremTime4</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>boxTremTime5</zorder>
   <zorder>boxTremTime6</zorder>
   <zorder>boxTremTime7</zorder>
   <zorder>boxTremLastTime4</zorder>
   <zorder>boxTremLastTime1</zorder>
   <zorder>boxTremLastTime2</zorder>
   <zorder>boxTremLastTime6</zorder>
   <zorder>boxTremLastTime7</zorder>
   <zorder>boxTremLastTime3</zorder>
   <zorder>boxTremLastTime5</zorder>
   <zorder>boxTremSD4</zorder>
   <zorder>boxTremSD1</zorder>
   <zorder>boxTremSD2</zorder>
   <zorder>boxTremSD6</zorder>
   <zorder>boxTremSD7</zorder>
   <zorder>boxTremSD3</zorder>
   <zorder>boxTremSD5</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
